portfolio/src/main/webapp/professional.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Carolyn's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+     <ul>
```

Comment:
This is great semantic html! You're hiding the bullets in css so it doesn't look like a list, but for accessibility reasons, this is best represented as one. (Screen readers will read "List with 5 items. First, link to home. Second, like to About Me..." Awesome!